### DISCUSSION TITLE
How to set an order for my seeders [V4.1]

### DISCUSSION THREAD

**Discussion by ymsaadi** [adonisjs/core#2632] [Help] [answered]
Hello, 

I'm trying to run some seed files for my database. There are a lot of relationships between my tables and I would like to set some kind of execution order. How do I do that ?

Thank you in advance.

**Comment by aayush123** [comment, marked as answer]
You can rename the seeder files and prefix numbers to order them. Eg:
`0_first_seeder.ts`
`1_second_seeder.ts`

**Reply by ymsaadi** [reply]
Oh, so the seeders get executed in alphabetical order ? I'll try that, thank you.
